easybuild/easyconfigs/h/HapCUT2/HapCUT2-v1.3.3-GCC-12.3.0.eb

```diff
@@ -0,0 +1,75 @@
+easyblock = 'MakeCp'
+name = 'HapCUT2'
+version = 'v1.3.3'
+
+homepage = 'https://github.com/vibansal/HapCUT2/'
+description = """
+HapCUT2 is a maximum-likelihood-based tool for assembling haplotypes from DNA sequence reads,
+designed to "just work" with excellent speed and accuracy.
+We found that previously described haplotype assembly methods are specialized for specific read technologies or
+protocols,
+with slow or inaccurate performance on others.
+With this in mind, HapCUT2 is designed for speed and accuracy across diverse sequencing technologies,
+including but not limited to:
+
+    - NGS short reads (Illumina HiSeq)
+    - single-molecule long reads (PacBio and Oxford Nanopore)
+    - Linked-Reads (e.g. 10X Genomics, stLFR or TELL-seq)
+    - proximity-ligation (Hi-C) reads
+    - high-coverage sequencing (>40x coverage-per-SNP) using above technologies
+    - combinations of the above technologies (e.g. scaffold long reads with Hi-C reads)
+"""
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+
+# source_urls = ['https://github.com/vibansal/HapCUT2/archive/refs/tags/']
+# sources = ['%(version)s.tar.gz']
+
+download_instructions = """
+  The last version tag v1.3.3 does not include last modification that make it work.
+  So we download source from master branch at the commit where I tested.
+"""
+sources = [{
+    'filename': '%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/vibansal/',
+        'repo_name': 'HapCUT2',
+        'commit': '0eb7075',
+        'recursive': True,
+        'keep_git_dir': True,
+    },
+}]
```

Comment:
is it really needed to do a git checkout here? why not download the commit tarball directly?
https://github.com/vibansal/HapCUT2/archive/0eb7075.tar.gz

Reply:
I copied the example from the documentation, and thought that was the right way to do it.

Reply:
we try to avoid it because it prevents proper checksumming.
it seems perfectly possible to use the tarball in this case as are there are no git submodules.